src/core/Akka/Routing/RouterConfig.cs

```diff
@@ -145,13 +145,20 @@ public abstract class Group : RouterConfig, IEquatable<Group>
         /// <param name="routerDispatcher">TBD</param>
         protected Group(IEnumerable<string> paths, string routerDispatcher) : base(routerDispatcher)
         {
-            Paths = paths;
```

Comment:
This is the real fix here. Needed to stop using the deprecated `Paths` property so the router will fall back to calling the `GetPaths(ActorSystem)` method instead, which does the right thing. Created a protected `_paths` property to hold onto the value that will be used inside that method in most built-in routers, which is identical to what the JVM has.